### PR TITLE
Update collaborate-in-site.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed from Azure org settings to External collaboration settings

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2921

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/solutions/collaborate-in-site.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This video shows the configuration steps described in this document.</br>
 
 > [!VIDEO https://www.microsoft.com/videoplayer/embed/RE44Llg?autoplay=false]
 
-## Azure Organizational relationships settings
+## Azure External Collaboration settings
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/collaborate-in-site.md]:
```suggestion
## Azure external collaboration settings
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**